gaskij/rpicampusmap
# Created "not found" and "info" page components, added as routes

## Description
Created a not found page and added its route to the main source. Made the not found page the default page in case a page name isn't specified.

Created an info page and added its route to the main source.

Fixes #(issue name)
Not Found page added, no issue
## Solution
(What did you do to fix this issue?)
N/A
## Known Issues
(Are there any knew issues caused or exposed by your changes, or things you would like clarified before merging?)
N/A
## Testing Procedures
(What should reviewers do to validate your changes? What steps should be taken and what is the expected outcome?)
Go to any unspecified page and check if not found page is visible
Using the campus map to navigate to the info page, check if correct information appears (LATER FEATURE)

## Checklist for author
